src/test/daily_mode.test.ts

```diff
@@ -28,5 +38,117 @@ describe("Daily Mode", () => {
       expect(pkm.level).toBe(20);
       expect(pkm.moveset.length).toBeGreaterThan(0);
     });
+    expect(game.scene.getModifiers(MapModifier).length).toBeGreaterThan(0);
+  });
+});
+
+//*
+// Need to figure out how to properly start a battle
+// Need to fix eviolite - test keeps insisting it is not in loot table, even though Mini Black Hole (which is using the exact same condition) is
+describe("Shop modifications", async () => {
+  let phaserGame: Phaser.Game;
+  let game: GameManager;
+
+  beforeAll(() => {
+    phaserGame = new Phaser.Game({
+      type: Phaser.HEADLESS,
+    });
+  });
+  beforeEach(() => {
+    game = new GameManager(phaserGame);
+
+    game.override
+      .startingWave(9)
+      .startingBiome(Biome.ICE_CAVE) // Will lead to Snowy Forest with randomly generated weather
+      .battleType("single")
+      .startingLevel(100) // Avoid levelling up
+      .enemyLevel(1000) // Avoid opponent dying before game.doKillOpponents()
+      .disableTrainerWaves()
+      .moveset([Moves.KOWTOW_CLEAVE])
+      .enemyMoveset(Moves.SPLASH);
+    game.modifiers.addCheck("EVIOLITE");
+    game.modifiers.addCheck("MINI_BLACK_HOLE");
+  });
+
+  afterEach(() => {
+    game.phaseInterceptor.restoreOg();
+    game.modifiers.clearChecks();
+  });
+
+  it("should not have Eviolite and Mini Black Hole available in Classic if not unlocked", async () => {
+    await game.classicMode.runToSummon();
```

Comment:
```suggestion
    await game.classicMode.startBattle();
```
This wouldn't have worked before I changed tests to use `SET` battle style by default lol.